src/classes/ACCT_AdministrativeNameRefresh_TEST.cls

```diff
@@ -32,7 +32,7 @@
 * @description Tests class for refreshing Administrative Account Names.
 */
 @isTest
-public class ACCT_AdministrativeNameRefresh_TEST {
+private class ACCT_AdministrativeNameRefresh_TEST {
```

Comment:
Changed access modifier to private, remaining changes to this class are variable naming and spacing to follow EDA style guide.